compiler/src/dotty/tools/dotc/transform/Mixin.scala

```diff
@@ -148,6 +148,10 @@ class Mixin extends MiniPhase with SymTransformer { thisPhase =>
           val setter = makeTraitSetter(decl.asTerm)
           decls1.enter(setter)
           modified = true
+        if decl.isAllOf(PrivateAccessor) then
```

Comment:
Since Private things become non-private during this transform, shouldn't we look `atPhase(thisPhase)(decl.isAllOf(PrivateAccessor))` instead?

Reply:
We are at `thisPhase` anyway when we do the transform.